README.rst

```diff
@@ -1525,6 +1525,7 @@ Once you have created the models, add the following to your ``settings.py``:
     GEO_DEVICELOCATION_MODEL = 'sample_geo.DeviceLocation'
     CONNECTION_CREDENTIALS_MODEL = 'sample_connection.Credentials'
     CONNECTION_DEVICECONNECTION_MODEL = 'sample_connection.DeviceConnection'
+    CONNECTION_COMMAND_MODEL = 'sample_connection.Command'
```

Comment:
BTW, the README doesn't mention this feature at all, we should fix this.

We should mention this feature in https://github.com/openwisp/openwisp-controller#connection-app, but I also think we could add a dedicated section which gives an overview and shows some screenshots or a GIF (I would host the GIF on an external server this time). Later on we can turn this into a dedicated page on https://openwisp.io/docs/.